src/Constants/Constants.tsx

```diff
@@ -0,0 +1,9 @@
+export const authEndpoint = "https://accounts.spotify.com/authorize"
```

Comment:
The `x` part of .tsx means that the file contains JSX, this should just be a .ts file